app/javascript/availability/index.js

```diff
@@ -610,6 +614,20 @@ const availability = {
 
     isArchivalMaterial(holding) {
         return (['UP-SPECCOL'].includes(holding.libraryID) && !availability.isMoved(holding.homeLocationID));
+    },
+
+    isHoldButton(holdings) {
```

Comment:
```suggestion
    isHoldable(holdings) {
```

is that better?

or maybe `showHoldButton` (wish we could have question marks). what you have is probably fine

Reply:
I like `showHoldButton` 👍